test/test.transporters.js

```diff
@@ -71,6 +71,17 @@ describe('Transporters', function() {
     assert.equal(req.body, null);
   });
 
+it('should return errors as instances of Error', function (done) {
+    var google = require('../lib/googleapis');
+    var pagespeed = google.pagespeedonline('v1');
+
+    // intentional grammar mistake to trigger the error
+    pagespeed.pagespeedapi.runpagespeed({url: 'http://gogle.con'}, function (err) {
```

Comment:
Is this making a real request? None of the unit tests should depend on network access.